			   READLINE PATCH REPORT
			   =====================

Readline-Release: 8.2
Patch-ID: readline82-008

Bug-Reported-by:
Bug-Reference-ID:
Bug-Reference-URL:

Bug-Description:

Add missing prototypes for several function declarations.

Patch (apply with `patch -p0'):

Upstream-Status: Submitted [https://ftp.gnu.org/gnu/readline/readline-8.2-patches/]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

*** ../readline-8.2-patched/text.c	Wed Oct 27 11:03:59 2021
--- text.c	Thu Nov 16 16:24:58 2023
***************
*** 1765,1770 ****
  #if defined (READLINE_CALLBACKS)
  static int
! _rl_char_search_callback (data)
!      _rl_callback_generic_arg *data;
  {
    _rl_callback_func = 0;
--- 1765,1769 ----
  #if defined (READLINE_CALLBACKS)
  static int
! _rl_char_search_callback (_rl_callback_generic_arg *data)
  {
    _rl_callback_func = 0;
*** ../readline-8.2-patched/bind.c	Wed Feb  9 11:02:22 2022
--- bind.c	Thu Nov 16 16:25:17 2023
***************
*** 1168,1174 ****
  
  static int
! parse_comparison_op (s, indp)
!      const char *s;
!      int *indp;
  {
    int i, peekc, op;
--- 1168,1172 ----
  
  static int
! parse_comparison_op (const char *s, int *indp)
  {
    int i, peekc, op;
*** ../readline-8.2-patched/rltty.c	Fri Feb 18 11:14:22 2022
--- rltty.c	Thu Nov 16 16:25:36 2023
***************
*** 81,86 ****
     to get the tty settings. */
  static void
! set_winsize (tty)
!      int tty;
  {
  #if defined (TIOCGWINSZ)
--- 81,85 ----
     to get the tty settings. */
  static void
! set_winsize (int tty)
  {
  #if defined (TIOCGWINSZ)

*** ../readline-8.2/patchlevel	2013-11-15 08:11:11.000000000 -0500
--- patchlevel	2014-03-21 08:28:40.000000000 -0400
***************
*** 1,3 ****
  # Do not edit -- exists only for use by patch
  
! 7
--- 1,3 ----
  # Do not edit -- exists only for use by patch
  
! 8
